css/user-search.styl

```diff
@@ -0,0 +1 @@
+@import "default.css"
```

Comment:
I think it may be worth changing up this line to `@import "../node_modules/react-select/dist/default.css"` and then the css import lines in the shell scripts can be just `--include-css  \` so the css is all bundled in the same file